checkov/terraform/checks/resource/azure/CognitiveServicesRestrictOutboundNetwork.py

```diff
@@ -0,0 +1,32 @@
+from typing import Dict, List, Any
+
+from checkov.common.models.enums import CheckCategories, CheckResult
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class CognitiveServicesRestrictOutboundNetwork(BaseResourceCheck):
+    def __init__(self):
+        name = "Ensure that Cognitive Services accounts restrict outbound network traffic."
+        id = "CKV_AZURE_247"
+        supported_resources = ('azurerm_cognitive_account', )
+        categories = (CheckCategories.NETWORKING, )
+        super().__init__(
+            name=name,
+            id=id,
+            categories=categories,
+            supported_resources=supported_resources,
+        )
+
+    def scan_resource_conf(self, conf: Dict[str, List[Any]]) -> CheckResult:
+        if conf.get("kind", [""])[0].lower() != 'openai':
```

Comment:
do we want this part or should it be for all kinds?

Reply:
Just OpenAI. This is a run check we are mirroring